pkg/kube/multicluster/secretcontroller.go

```diff
@@ -87,7 +88,7 @@ type Controller struct {
 }
 
 // NewController returns a new secret controller
-func NewController(kubeclientset kube.Client, namespace string, clusterID cluster.ID, meshWatcher mesh.Watcher) *Controller {
+func NewController(kubeclientset kube.Client, namespace string, clusterID cluster.ID, meshWatcher mesh.Watcher, configOverrides ...func(*rest.Config)) *Controller {
```

Comment:
When the sidecar locates the remote cluster and connects to the istiod, it will use the client which  belonging to the remote cluster to auth the connection, The incoming XDS requests per second may be limited by the QPS if the `features.RequestLimit` is higher than the QPS.

Reply:
I missed it will create kubeclient for remote clusters.